src/index.html

```diff
@@ -11,12 +11,98 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="page">
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo"
+          class="logo__img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
```

Comment:
Ensure that the `data-qa` attribute is necessary and used correctly. If it's for testing purposes, make sure it's documented or used consistently across the project.